src/coreclr/src/jit/morph.cpp

```diff
@@ -6000,8 +6000,10 @@ GenTree* Compiler::fgMorphField(GenTree* tree, MorphAddrContext* mac)
 
         bool addExplicitNullCheck = false;
 
-        // Implicit byref locals are never null.
-        if (!((objRef->gtOper == GT_LCL_VAR) && lvaIsImplicitByRefLocal(objRef->AsLclVarCommon()->GetLclNum())))
+        // Constant strings are never null.
```

Comment:
Nit: we don't put comments in the middle, could you please attach `// Implicit byref locals are never null.` to `// Constant strings are never null. Or refactor the whole block as:
```
 bool addExplicitNullCheck = true;
if (objRef->OperIs(GT_CNS_STR))
{
  addExplicitNullCheck = false;
}
else if (objRef->OperIs(GT_LCL_VAR) && lvaIsImplicitByRefLocal(objRef->AsLclVarCommon()->GetLclNum())
{
  addExplicitNullCheck = false;
}
```
etc...

Reply:
oops, didn't notice this comment, addressed. thanks!